infra/gcp/bash/ensure-conformance-storage.sh

```diff
@@ -20,7 +20,6 @@
 # dedicated Google groups.
 #
 # The buckets will have a layout which is readable by testgrid:
-# (See also: https://github.com/kubernetes/test-infra/tree/master/gubernator#gcs-layout)
```

Comment:
This one's actually still needed. We should move it out of this location but the layout info described here is still correct.